### PR TITLE
Home work 10

### PR DESCRIPTION
**Pull request by centora**

### CODE REVIEW

File: server-app/package.json [hunk 1]
```diff
@@ -0,0 +1,15 @@
+{
+  "name": "server-client-app",
+  "version": "1.0.0",
+  "description": "",
+  "main": "server.js",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1",
+    "start": "node server.js",
+    "server": "node server.js",
+    "stop": "ps -ef | grep \"node\" | awk '{print $2}' | xargs kill -9",
```
Comment (iliatcymbal) [inline server-app/package.json]: which node process will be terminated if you have few node processes?

File: server-app/server.js [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+      res.setHeader('Content-Type', 'text/html');
+      if(err) {
+        return res.end('<strong>Error</strong>');
+      }
+
+      const displayDate = data.replace('</body>', `<p>Today is: <em>${date.toLocaleDateString()}</em></p></body>`)
+      res.end(displayDate);
+    });
+    return
+  } else if (isJS) {
```
Comment (iliatcymbal) [inline server-app/server.js]: why you need else here?

File: server-app/server.js [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+        return res.end('<strong>Error</strong>');
+      }
+      res.end(data);
+    });
+    return
+  }
+
+  fs.readFile(fileName, (err, data) => {
+    if(err) {
+      return res.end('<strong>Error</strong>');
```
Comment (iliatcymbal) [inline server-app/server.js:35]: which content type for this response?

**Review by iliatcymbal — APPROVED** [review]
see my notes

File: server-app/server.js (hunk 3 above)
Comment (centora) [reply server-app/server.js:35]: I thought we don't need to use any headers or content type  if we want to get image file

**Pull request closed without merging**